packages/accounts/src/kernel-zerodev/middleware/paymaster.ts

```diff
@@ -29,14 +29,25 @@ export const zeroDevPaymasterAndDataMiddleware = <
       return struct;
     },
     paymasterDataMiddleware: async (struct) => {
-      struct.preVerificationGas = BigInt("100000");
-      struct.verificationGasLimit = BigInt("1000000");
-      struct.callGasLimit = BigInt("55000");
+      const preVerificationGas = BigInt("100000");
+      const verificationGasLimit = BigInt("1000000");
+      const callGasLimit = BigInt("55000");
```

Comment:
You removed it here from the struct, but if there is no paymaster, is it still being set? And if so, why specify it then at two places?

Reply:
The idea is to use either paymaster or gasEstimator. But don't want to include them in struct if paymaster fails.